examples/docs/es/select.md

```diff
@@ -317,11 +319,12 @@ You can clear Select using a clear icon.
 ```
 :::
 
-### Basic multiple select
+### Selección múltiple básica
+
+Selección multiple utiliza tags para mostrar las opciones seleccionadas.
 
-Multiple select uses tags to display selected options.
+:::demo Configure el atributo `multiple` para `el-select` para habilitar el modo múltiple. En este caso, el valor del `v-model` será un array de opciones seleccionadas. By default the selected options will be displayed as Tags. You can collapse them to a text by using `collapse-tags` attribute.
```

Comment:
Sorry I forgot something:
> By default the selected options will be displayed as Tags. You can collapse them to a text by using `collapse-tags` attribute.

This last sentence is added a couple of days ago, so could you send another PR for `collapse-tags`? Sorry for the trouble.